src/client/public/scripts/dash/dash.ts

```diff
@@ -1,6 +1,7 @@
 import {html, render} from '../../../../../node_modules/lit-html/lit-html.js';
 import * as api from '../../../../types/api';
-import {JSONAPIResponse, LastKnownResponse, PullRequest} from '../../../../types/api';
+
+import {JSONAPIResponse, LastKnownResponse, PullRequest, SWClientMessage} from '../../../../types/api';
```

Comment:
This has 2 imports form `api` here. Do you mind cleaning that up here? Either the second style where we specify exactly what we want from the API, or the first where we import everything and use `api.*` everywhere.